### DISCUSSION TITLE
Add `hide_background` option for envmaps

### DISCUSSION THREAD

**Discussion by georgy95** [mitsuba-renderer/mitsuba3#819] [Feature requests] [answered]
During model rendering, I'd like to use the environment map for lighting purposes only (as an injection). 

My goal is to render a 3D model in 360 view, without any background, which is automatically set by the envmap.

As is stands, we need to add additional 3D objects into the scene to fill the background space.

**Comment by njroussel** [comment, marked as answer]
Hi @skiler07 

Most integrators have a parameter `hide_emitters` ([example in `direct`](https://mitsuba.readthedocs.io/en/stable/src/generated/plugins_integrators.html#direct-illumination-integrator-direct)). When enabling this, it should behave as you describe -- of course this assumes the only directly visible source of light in your scene is the environment map.

**Reply by Speierers** [reply]
Although I think this option is still missing in some integrators (e.g. `prb`).
